pkg/beaconproxy/mongodb.go

```diff
@@ -47,6 +47,7 @@ func (r *repo) CreateIndexes() error {
 	// set desired indexes
 	indexes := []mgo.Index{
 		{Key: []string{"-score"}},
+		{Key: []string{"src", "dst", "fqdn", "src_network_uuid", "dst_network_uuid"}, Unique: true},
```

Comment:
In #690 , we are changing the keying to be src-fqdn and cutting out the dst (i.e., proxy) as part of the keying since it isn't really relevant for analysis. Can we please cut dst and dst_network_uuid from this key and just have it be "src" and "fqdn"?

Reply:
Let's revert the `beaconProxy` index commit from this PR and apply the change only in #690.